lib/airbrake.js

```diff
@@ -183,11 +186,17 @@ Airbrake.prototype.notify = function (err, cb) {
     }
   });
 
-  if (exit) {
+  var notice = this.notifyJSON(err);
+
+  this.filters.forEach(function (filter) {
+    notice = filter(notice);
```

Comment:
If there are multiple filters and one of them filters out the notice, then next filter will receive `null` instead of `notice`, which will result in a crash. Could you add a test covering this scenario?